model/integration_action.go

```diff
@@ -530,3 +532,136 @@ func DoPostActionRequestFromJson(data io.Reader) *DoPostActionRequest {
 	json.NewDecoder(data).Decode(&o)
 	return o
 }
+
+func (r *OpenDialogRequest) IsValid() error {
```

Comment:
Everything in this file seems to agree with the documentation, but I wonder if we are being so strict in the code, and if adding these restrictions may break working integrations.